.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,23 +4,26 @@ about: Create a report to help us improve
 
 ---
 
-**Checklist:**
+**Before you submit:**
 
-- [ ] I have checked stackoverflow for this bug and there are no answers or the question does not exist
-- [ ] I have searched through the issue list for related bugs or issues
+- [ ] Please search through the existing issues (both open AND closed) to see if your issue has been discussed before. Github issue search can be used for this: https://github.com/imgix/drift/issues?utf8=%E2%9C%93&q=is%3Aissue
+- [ ] Please ensure the problem has been isolated and reduced. This link explains more: http://css-tricks.com/6263-reduced-test-cases/
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+A clear and concise description of what the bug is. Please strive to reach the **root problem** of your issue to avoid the XY problem. See more: https://meta.stackexchange.com/questions/66377/what-is-the-xy-problem
 
 **To Reproduce**
+A bug is a _demonstrable problem_ that is scause by the code in the repository. Thus, the contributors need a way to reproduce your issue - if we can't reproduce your issue, we can't help you! Also, please be as detailed as possible!
```

Comment:
s/scause/caused, and let's switch the exclamation point in the last sentence here to a period.